MmSupervisorPkg/Core/Request/UpdateCommBuffer.c

```diff
@@ -0,0 +1,174 @@
+/** @file
+  PI SMM MemoryAttributes support

```

Comment:
File description looks irrelevant.

Reply:
Updated. (and some other files as well.)